### DISCUSSION TITLE
how to upgrade the milvus 0.10.0 to 2.0

### DISCUSSION THREAD

**Discussion by tom729** [milvus-io/milvus#20800] [Q&A and General discussion]
we start the service at 2020, so how to upgrade it to the last version, we can't find any doc about the case

**Comment by yhmo** [comment]
v0.10.x and v1.x cannot directly upgrade to v2.x.
We provide a tool to export data of v0.10.x/v1.x and insert the data into v2.x:
https://github.com/milvus-io/milvus-tools

Note: only the original data is exported, not including index data.
